src/harness/fourslash.ts

```diff
@@ -3384,7 +3384,7 @@ ${code}
         }
 
         // @Filename is the only directive that can be used in a test that contains tsconfig.json file.
-        if (containTSConfigJson(files)) {
+        if (files.some(f => f.fileOptions.Filename === "tsconfig.json")) {

```

Comment:
Should it consider uppercase characters as well?

Reply:
Good catch -- we were doing that elsewhere but not here.